### ISSUE TITLE
Compilation errors occur  while integrating converted model to example.cc on linux,  

### ISSUE THREAD

**Issue by Zhengtq** [XiaoMi/mace#147]
I have converted a tf_mobilenet_v2 model to through mace,below is my deployment file:
library_name: mobilenet_v2
target_abis: [host]
model_graph_format: code
model_data_format: file
models:
  mobilenet_v2_host:
    platform: tensorflow
    model_file_path: /home/z/graph.pb
    model_sha256_checksum: 289c2b9d1b12bcf80db017478f8fdced77252f46896a1fc54cede89e1f9a6c74
    subgraphs:
      - input_tensors:
          - validate_input/imgs 
        input_shapes:
          - 1,320,320,3
        output_tensors:
          -  MobilenetV2/Logits/Zoutput
        output_shapes:
          - 1,2
    runtime: cpu

I integrate my converted tf_mobilenet_v2 model to the example.cc(mace/examples/example.cc) on my linux computer. I have linked all possible libs and headers(libmace.a  libmace.so  mobilenet_v2.a,mace_engine_factory.h  mace.h  mace_runtime.h  mobilenet_v2_host.h ). And I use "MODEL_GRAPH_FORMAT_CODE" in my code file. However, there are compilation errors when I compile the code. Below are part of the errors:

../lib/ztq_mobv2_8_1.a(tensor99.o): In function `mace::mobilenet_v2_host::CreateTensor99(mace::ConstTensor*)':
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x2e): undefined reference to `mace::logging::LogMessage::MinVLogLevel()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x189): undefined reference to `mace::logging::LogMessage::MinVLogLevel()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x272): undefined reference to `google::protobuf::internal::fixed_address_empty_string[abi:cxx11]'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x34d): undefined reference to `mace::logging::LogMessage::MinVLogLevel()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x410): undefined reference to `mace::logging::LogMessage::MinVLogLevel()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x437): undefined reference to `mace::logging::LogMessage::LogMessage(char const*, int, int)'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x467): undefined reference to `mace::logging::LogMessage::~LogMessage()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x490): undefined reference to `mace::logging::LogMessage::MinVLogLevel()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x4be): undefined reference to `mace::logging::LogMessage::LogMessage(char const*, int, int)'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x50b): undefined reference to `mace::logging::LogMessage::~LogMessage()'
tensor99.cc:(.text._ZN4mace17mobilenet_v2_host14CreateTensor99EPNS_11ConstTensorE+0x6ae): undefined reference to `mace::logging::LogMessage::~LogMessage()' 

I shows that there maybe some libs that I forget linking. Can anyone tells me how to solve it.

**Comment by liyinhgqw** [comment]
I thinks they are all libraries that you need. What is your link option? 
Do you put libmace.a after mobilenet_v2.a?
And you cannot link both libmace.a and libmace.so
